src/NewsService.Data/NewsRepository.cs

```diff
@@ -41,11 +63,11 @@ public async Task<bool> EditAsync(Guid newsId, JsonPatchDocument<DbNews> patch)
       return true;
     }
 
-    public async Task<Guid?> CreateAsync(DbNews dbNews)
+    public async Task<Guid> CreateAsync(DbNews dbNews)
     {
-      if (dbNews == null)
+      if (dbNews is null)
       {
-        return null;
+        return default;

```

Comment:
default is Guid.Empty, why you do not wont to return null?

Reply:
becouse it is not nullable type